resolver/src/main/java/io/netty/resolver/InetSocketAddressResolver.java

```diff
@@ -30,7 +30,7 @@
  */
 public class InetSocketAddressResolver extends AbstractAddressResolver<InetSocketAddress> {
 
-    private final NameResolver<InetAddress> nameResolver;
+    protected final NameResolver<InetAddress> nameResolver;
```

Comment:
consider adding a protected final accessor method instead of directly exposing the member variable, or make it package private for now.